vktrace/vktrace_layer/vktrace_lib_trim.cpp

```diff
@@ -376,11 +376,11 @@ void initialize() {
         getTraceTriggerOptionString(enum_trim_trigger::frameCounter);
     if (trimFrames != nullptr) {
         uint32_t numFrames = 0;
-        if (sscanf(trimFrames, "%llu,%lu", &g_trimStartFrame, &numFrames) ==
+        if (sscanf(trimFrames, "%" PRIx64 ",%" PRIx32, &g_trimStartFrame, &numFrames) ==
```

Comment:
I think PRIu64 and PRIu32 would make more sense here, integers instead of hex values.